chain/consensus/mir/mempool/handlers/batches.go

```diff
@@ -0,0 +1,84 @@
+package handlers
+
+import (
+	"github.com/filecoin-project/lotus/chain/consensus/mir/mempool/types"
+	"github.com/filecoin-project/mir/pkg/dsl"
+	mpdsl "github.com/filecoin-project/mir/pkg/mempool/dsl"
+	"github.com/filecoin-project/mir/pkg/pb/mempoolpb"
+	"github.com/filecoin-project/mir/pkg/pb/requestpb"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+type State struct {
+	*types.State
+	NewTxIDs []t.TxID
```

Comment:
Does this come from an old version of the code or did we agree to have a stateless pool only in the next iteration, when we also have client watermarks?
@dnkolegov ?

Reply:
Yes, this came from the initial implementation.

Reply:
Removed